tasklist/client/src/v2/TaskDetails/index.test.tsx

```diff
@@ -260,6 +263,7 @@ describe('<Task />', () => {
       http.post(
         '/v2/user-tasks/:userTaskKey/completion',
         () => {
+          taskState = 'COMPLETED';
```

Comment:
Do not mutate the state like this, it's hard to follow what's happening
If you call `  http.get('/v2/user-tasks/:userTaskKey'` with the different state will overwrite the previous mock